aws-athena/views/reporting-vw_most_recent_boundary.sql

```diff
@@ -0,0 +1,168 @@
+-- View that always contains the most recent political boundaries the
+-- data department has ingested, by 10-digit parcel.
+
+WITH parcel AS (
+    SELECT
+        parcel.pin10,
+        parcel.town_code AS township_code,
+        parcel.year AS parcel_year,
+        ST_POINT(parcel.x_3435, parcel.y_3435) AS geom
+    FROM {{ source('spatial', 'parcel') }} AS parcel
+    WHERE parcel.year
+        = (
+            SELECT MAX(parcel.year)
+            FROM {{ source('spatial', 'parcel') }} AS parcel
+        )
+        AND parcel.town_code IS NOT NULL
```

Comment:
**question (non-blocking):** What's the purpose of this line?

Reply:
Resolved offline. In the parcel file, PINs with a null town code don't actually seem to exist.